subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -3728,6 +3728,43 @@ static bool lwm2m_engine_path_included(uint8_t code, bool bootstrap_mode)
 	return true;
 }
 
+static int lwm2m_engine_observation_handler(struct lwm2m_message *msg, int observe, uint16_t accept)
+{
+	int r;
+
+	if (observe == 0) {
+		/* add new observer */
+		r = coap_append_option_int(msg->out.out_cpkt, COAP_OPTION_OBSERVE,
+					   OBSERVE_COUNTER_START);
+		if (r < 0) {
+			LOG_ERR("OBSERVE option error: %d", r);
+			return r;
+		}
+
+		r = engine_add_observer(msg, msg->token, msg->tkl, accept);
+		if (r < 0) {
+			LOG_ERR("add OBSERVE error: %d", r);
+		}
+
```

Comment:
could you please be consistent in whether you have a empty line or not right before the `} else ... {`  .
Right now you mix like this:
```
if (...) {
    {
    }
<empty line>
} else if (...) {
    {
    }
} else {
   ...
}
```

```suggestion
```

Reply:
Thanks for good point I will clean those empty lines.